Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,15 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
```

Comment:
Sure, I can actually picture more people feeling "violated" when they see this... Removed.